docker-storage-setup.conf

```diff
@@ -4,9 +4,9 @@
 STORAGE_DRIVER=devicemapper
 
 # A set of extra options that should be appended to the generated
-# DOCKER_STORAGE_OPTIONS string. These options will be passed to the Docker
+# STORAGE_OPTIONS string. These options will be passed to the Docker
 # daemon as-is and should be valid Docker storage options.
```

Comment:
I plan on other pull requests to change the Documentation and function names,  Want to take it one step at a time.